scripts/install-k8s-extension.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
```

Comment:
I'd highly recommend `set -aueo pipefail` here (in another PR or whenever)

And the `u` in that will complain about `$SUBSCRIPTION` (and a few of the other variables) most likely 🤔